### PR TITLE
7200 widgets source

### LINKED ISSUE #7200: Store sourceId for widgets
Since we are supporting dataviews having an analysis as a type of source (https://github.com/CartoDB/cartodb.js/issues/1212), we now need to store the `sourceId`  (the id of the analysis) whenever we create a widget.

/cc @alonsogarciapablo

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #7200.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
Retest this, please.

**Comment by juanignaciosl** [comment]
@gfiorav CR this, please.

**Comment by jvillarf** [comment]
Retest this, please.

File: db/migrate/20160425083000_widgets_source_id.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  up do
+    alter_table(:widgets) do
+      add_column :source_id, :text
```
Comment (gfiorav) [inline db/migrate/20160425083000_widgets_source_id.rb]: Why `:text`?

**Comment by gfiorav** [comment]
Made a minor coment. Also, tests seem not to be passing.

After that it's 📦 for me!

File: db/migrate/20160425083000_widgets_source_id.rb (hunk 1 above)
Comment (juanignaciosl) [inline db/migrate/20160425083000_widgets_source_id.rb]: It's not another database column but an analysis "natural id".

**Comment by juanignaciosl** [comment]
@gfiorav tests passed 👍

**Comment by gfiorav** [comment]
What's wrong with `continuous-integration/travis-ci/push`?

📦 it if it's unimportant!

**Pull request merged**